### PR TITLE
Add a prerequisites section for building docs

### PR DESCRIPTION
**Pull request by shivaram**
This puts all the install commands that need to be run in one section instead of being spread over many paragraphs

cc @rxin

### CODE REVIEW

**Comment by rxin** [comment]
Thanks - merging.

File: docs/README.md [hunk 1]
```diff
@@ -8,6 +8,16 @@ Read on to learn more about viewing documentation in plain text (i.e., markdown)
 documentation yourself. Why build it yourself? So that you have the docs that corresponds to
 whichever version of Spark you currently have checked out of revision control.
 
+## Prerequisites
+The Spark documenation build uses a number of tools to build HTML docs and API docs in Scala, Python
+and R. To get started you can run the following commands
+
+    $ sudo gem install jekyll
```
Comment (srowen) [inline docs/README.md]: This is mostly redundant with existing docs below though ... ? delete one of them

Comment (shivaram) [inline docs/README.md]: @rxin can probably add more, but I meant this to be redundant in the sense this section had a bunch of commands you could copy-paste while the succeeding sections had the rationale / breakdown of what was required when.

Comment (srowen) [inline docs/README.md]: OK no problem. It's hardly a big deal; was more worrying about later updating one bit but not the other but, it's an ancillary doc that's not frequently updated.

Comment (rxin) [inline docs/README.md]: I will reword README in a bit to make it more consistent.

**Pull request closed without merging**